src/Resources/skeleton/doctrine/Repository.tpl.php

```diff
@@ -19,27 +19,21 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, <?= $entity_class_name; ?>::class);
     }
 
-    /**
-     * @throws ORMException
-     * @throws OptimisticLockException
-     */
     public function add(<?= $entity_class_name ?> $entity, bool $flush = false): void
     {
-        $this->_em->persist($entity);
+        ($em = $this->getEntityManager())->persist($entity);
```

Comment:
That seems more fancy than it needs to be. I'd prefer `$this->getEntityManager()` in both places.